modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/ProteinFeatures_conf.pm

```diff
@@ -851,6 +851,21 @@ sub pipeline_analyses {
       -parameters        => {
                               analyses => $self->o('protein_feature_analyses')
                             },
+      -flow_into         => {
+                              '-1' => ['StoreProteinFeatures_HighMem'],
+                            },
+    },
+    
+    {
+      -logic_name        => 'StoreProteinFeatures_HighMem',
+      -module            => 'Bio::EnsEMBL::Production::Pipeline::ProteinFeatures::StoreProteinFeatures',
+      -analysis_capacity => 10,
+      -batch_size        => 50,
+      -max_retry_count   => 1,
+      -parameters        => {
+                              analyses => $self->o('protein_feature_analyses')
+                            },
+      -rc_name           => '4GB',
```

Comment:
I can only see those resource classes in config, is `4GB` issued from parent class?

```   
'4GB_4CPU' => {'LSF' => '-q '.$self->o('production_queue').' -n 4 -M  4000 -R "rusage[mem=4000]"'},
'16GB_4CPU' => {'LSF' => '-q '.$self->o('production_queue').' -n 4 -M 16000 -R "rusage[mem=16000]"'},
'32GB_4CPU' => {'LSF' => '-q '.$self->o('production_queue').' -n 4 -M 32000 -R "rusage[mem=32000]"'},
```

Reply:
Yes.

Reply:
So there must be some duplicates around :-)